build.gradle

```diff
@@ -17,9 +17,9 @@ allprojects {
 
 subprojects {
     group = 'com.justyoyo.contrast'
-    version = '0.1.0-SNAPSHOT'
+    version = '0.1.1-SNAPSHOT'
 }
 
 task wrapper(type: Wrapper) {
-    gradleVersion = '1.12'
+    gradleVersion = '2.3'
```

Comment:
Your gradle wrapper version does not match your bin. Don't foget to run `gradle wrapper` after updating the task.